config/501-policy-webhook-configuration.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: defaulting.clusterimagepolicy.sigstore.dev
+webhooks:
+  - admissionReviewVersions:
+      - v1
+    clientConfig:
+      service:
+        name: webhook
+        namespace: cosign-system
+        path: /default-sigstore-dev-v1alpha1-clusterimagepolicy
+    failurePolicy: Fail
+    matchPolicy: Equivalent
+    name: defaulting.clusterimagepolicy.sigstore.dev
+    rules:
+      - apiGroups:
+          - sigstore.dev
+        apiVersions:
+          - v1alpha1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - clusterimagepolicies
```

Comment:
This will be automatically populated by the webhook when it adds the `caBundle`

Reply:
Oh cool, we didnt realize that knative was modifying the webhook configurations. Made the changes now!